src/hotspot/share/classfile/modules.cpp

```diff
@@ -559,6 +560,46 @@ void Modules::verify_archived_modules() {
   ModuleEntry::verify_archived_module_entries();
 }
 
+#if INCLUDE_CDS_JAVA_HEAP
+char* Modules::_archived_main_module_name = nullptr;
+#endif
+
+void Modules::dump_module_name() {
```

Comment:
This function should be `dump_main_module_name()`

Reply:
I've changed the function name.